gradle/libs.versions.toml

```diff
@@ -1,11 +1,11 @@
 [versions]
 # Platform expectations
-paper = "1.18.1-R0.1-SNAPSHOT"
+paper = "1.18.2-R0.1-SNAPSHOT"
 
 # Plugins
 plotsquared = "6.8.1"
 worldedit = "7.2.10"
-protocollib = "4.8.0"
+protocollib = "5.0.0-SNAPSHOT"

```

Comment:
Is there no release that serves our needs?

Reply:
Not yet, the future Wrapper for the chunk packet will be merge into the 5.0.0 snapshot.

Reply:
No idea :/ But in itself their addition/pr is not _necessary_ at the moment if it annoys you too much, the plugin works perfectly with the current workaround.

Reply:
If we don't use anything from v5 we can safely stay on 4.8.0